native/effektio/src/events/mod.rs

```diff
@@ -0,0 +1,13 @@
+use lazy_static::lazy_static;
+use tokio::runtime;
+
+lazy_static! {
+    static ref RUNTIME: runtime::Runtime =
+        runtime::Runtime::new().expect("Can't start Tokio runtime");
+}
```

Comment:
??? Why are you adding a second runtime here? Please don't do that, use the one from `api.rs`.
```suggestion
```

Reply:
Fixed.
https://github.com/effektio/effektio/pull/177/commits/a399579d28b8e10df1d8f1af4e438c173d57e425